src/lib/importers/LcmsOpenSciEdLegacyImporter.ts

```diff
@@ -56,33 +56,53 @@ export default class LcmsOpenSciEdLegacyImporter {
   async importBundle(ocxUrlString: string): Promise<OpenSciEdLegacyOcxBundle> {
     const ocxBundle = await this.findOrCreateBundle(ocxUrlString)
 
-    await ocxBundle.createNodesFromUnitHtml(db, ocxUrlString)
```

Comment:
I don't remember if we have tests for this function, but in that case we should add checks for the status updates.